Tests/CSRFTests/XCTestManifests.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+
+extension CSRFTests {
+    static let __allTests = [
+        ("testThadCSRFMiddlewareBlockInvalidToken", testThadCSRFMiddlewareBlockInvalidToken),
+        ("testThadCSRFMiddlewareBlockNoToken", testThadCSRFMiddlewareBlockNoToken),
+        ("testThatCSRFMiddlewareFailsWithNoSession", testThatCSRFMiddlewareFailsWithNoSession),
+        ("testThatCSRFMiddlewareReturnsToken", testThatCSRFMiddlewareReturnsToken),
+        ("testTokenRoundTripUsingHeader", testTokenRoundTripUsingHeader),
+        ("testTokenRoundTripUsingJSONBody", testTokenRoundTripUsingJSONBody),
+        ("testTokenRoundTripUsingMultipartBody", testTokenRoundTripUsingMultipartBody),
+    ]
+}
+
+#if !os(macOS)
```

Comment:
Don't wrap this in an `#if !os(macOS)` - it is just ignored by Swift on macOS and may potentially be used in the future

Reply:
This codes was autogenerated by SwiftPM (```swift test --generate-linuxmain```).
Therefore, I think that there is no need to modify it.